src/ObjLoading/Game/T6/AssetLoaders/AssetLoaderLeaderboard.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "AssetLoading/BasicAssetLoader.h"
+#include "AssetLoading/IAssetLoadingManager.h"
+#include "Game/T6/T6.h"
+#include "SearchPath/ISearchPath.h"
+
+namespace T6
+{
+    class AssetLoaderLeaderboard final : public BasicAssetLoader<AssetLeaderboard>
```

Comment:
Since you newly added this AssetLoader, you will have to register it in ObjLoaderT6.cpp